Find a functionally equivalent code:android.telephony.PhoneNumberUtils.isPotentialLocalEmergencyNumber:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * Checks if a given number might *potentially* result in a call to an
 * emergency service, for the country that the user is in. The current
 * country is determined using the CountryDetector.
 *
 * Specifically, this method will return true if the specified number
 * is an emergency number in the current country, *or* if the number
 * simply starts with the same digits as any emergency number for the
 * current country.
 *
 * This method is intended for internal use by the phone app when
 * deciding whether to allow ACTION_CALL intents from 3rd party apps
 * (where we're required to *not* allow emergency calls to be placed.)
 *
 * @param number the number to look up.
 * @param context the specific context which the number should be checked against
 * @return true if the specified number is an emergency number for a local country, based on the
 * CountryDetector.
 *
 * @see android.location.CountryDetector
 * @hide
 */

@@ -1,3 +1,3 @@
 {
-    return isLocalEmergencyNumberInternal(number, context, false);
+    return isPotentialLocalEmergencyNumber(context, getDefaultVoiceSubId(), number);
 }

